docs/installation-steps/DeployWithDockerOnLinuxMac.md

```diff
@@ -94,10 +57,48 @@ Useful commands:
 - **docker-compose down**
 
 
-5. Open your web-browser with an IP address of the deployed environment at port **8080**
+4. Open your web browser with an IP address of the deployed environment at port **8080**
 
 Use the following **login\pass** to access:
 * Default User: `default\1q2w3e`
 * Administrator: `superadmin\erebus`
 
 > ⚠️ Please change the admin password for better security
+
+## Optional Customisation
+
+1. Expose Docker Volumes to the file system
+
+>    * OPTIONAL: Set {vm.max_map_count} kernel setting before ReportPortal deploying with the following [Commands](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/docker.html#docker-cli-run-prod-mode)
```

Comment:
Please remove asterisk and empty spaces here.

Reply:
done